deployment/README.md

```diff
@@ -94,3 +94,66 @@ bash-4.4# ./scripts/infra apply
 ```
 
 This will attempt to apply the plan assembled in the previous step using Amazon's APIs.
+
+## Logs and Debugging
+
+
+### CloudWatch logs and Dashboard
+
+Navigate to [CloudWatch](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#dashboards:). Select the dashboard for the environment you're investigating. Adjust the dashboard to reflect the time period you're interested in using the 'Custom' time filter, if needed.
+
+The Request Latency graph shows the top 50% fastest response times, the top 90%, and the top 99%. Requests that take over 30 seconds to respond will time out.
+
+The Requests graph shows the number of requests being received at a certain time.
+
+The Database CPU graph shows the percentage of database CPU being used at a certain time.
+
+CloudWatch logs are divided into groups. You will most likely be interested in the ones named log[Environment]App and log[Environment]AppCLI.
```

Comment:
Consider explaining that logs with the`App` suffix are from web tasks and logs with the `AppCLI` suffix were generated by running Django management commands.

Reply:
Added